core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/WorkspaceMigrationActorSpec.scala

```diff
@@ -108,6 +108,8 @@ class WorkspaceMigrationActorSpec
       IO.pure {
         Seq(Operation.newBuilder.setDone(true).build)
       }
+
+    override def getTransferOperation(operationName: GoogleStorageTransferService.OperationName): IO[Operation] = IO.pure(Operation.newBuilder().build())
```

Comment:
Why did this need to be implemented?

Reply:
becuz it's the`trait` requires it, so the mock has to implement it

Reply:
Ahhhh I thought it was already mocked out in the `MockGoogleStorageTransferService`! I would prefer if this threw if it were called. Could you delete this line and add the following line to the `MockGoogleStorageTransferService` please?

```scala
 override def getTransferOperation(operationName: GoogleStorageTransferService.OperationName): F[Operation] = ???
```

Reply:
yea ofc!

Reply:
Sugestion in #1688 1688